test/api/v3/integration/tasks/challenges/POST-tasks_challenge_id.test.js

```diff
@@ -0,0 +1,131 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../../helpers/api-v3-integration.helper';
+import { v4 as generateUUID } from 'uuid';
+
+describe('POST /tasks/challenge/:challengeId', () => {
+  let user;
+  let guild;
+  let challenge;
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 1});
+    guild = await user.post('/groups', {
+      name: 'Test Public Guild',
+      type: 'guild',
+    });
+    challenge = await user.post('/challenges', {
+      name: 'Test Challenge',
+      shortName: 'TChallenge',
+      leader: user._id,
```

Comment:
This param gets ignored in the route. The leader is assigned by the user making the request.